content/reference/api/hub/latest.yaml

```diff
@@ -6,112 +6,104 @@ info:
   - name: authentication
     x-displayName: Authentication
     description: |
-      Most Docker Hub API endpoints require you to authenticate using your
-      Docker credentials before using them.
+      Most Docker Hub API endpoints require you to authenticate using your Docker credentials before using them.
 
-      Additionally, similar to the Docker Hub UI features, API endpoint responses may vary depending 
-      on your plan (Personal, Pro, or Team) and your account's permissions.
+      Additionally, similar to the Docker Hub UI features, API endpoint responses may vary depending on your plan (Personal, Pro, or Team) and your account's permissions.
 
       To learn more about the features available in each plan and to upgrade your existing plan, see [Docker Pricing](https://www.docker.com/pricing).
   - name: access-tokens
     x-displayName: Personal Access Tokens
     description: |
-      The Personal Access Token endpoints lets you manage personal access tokens. For more 
-      information, see [Access Tokens](https://docs.docker.com/security/for-developers/access-tokens/).
+      The Personal Access Token endpoints lets you manage personal access tokens. For more information, see [Access Tokens](https://docs.docker.com/security/for-developers/access-tokens/).
 
-      You can use a personal access token instead of a password in the [Docker CLI](https://docs.docker.com/engine/reference/commandline/cli/) 
-      or in the [Create an authentication token](#operation/PostUsersLogin) route to obtain a bearer 
-      token.
+      You can use a personal access token instead of a password in the [Docker CLI](https://docs.docker.com/engine/reference/commandline/cli/) or in the [Create an authentication token](#operation/PostUsersLogin) route to obtain a bearer token.
 
       ### Scopes
 
-      For each scope grouping (in this case "repo"), you only need to define 1 scope as any lower 
-      scopes are assumed. For example: If you define `repo:write`, the API assumes the scope of both 
-      `repo:read` *and* `repo:public_read` as well. If you were to define both `repo:write` *and* 
-      `repo:read`, then `repo:read` is assumed by `repo:write` and ignored.
+      For each scope grouping (in this case "repo"), you only need to define 1 scope as any lower scopes are assumed.
+      For example: If you define `repo:write`, the API assumes the scope of both `repo:read` *and* `repo:public_read` as well.
+      If you were to define both `repo:write` *and* `repo:read`, then `repo:read` is assumed by `repo:write` and ignored.
 
-      ***Treat your personal access token like your password and keep it secret. You cannot retrieve 
-      your token after it is generated.***
+      ***Treat your personal access token like your password and keep it secret. You cannot retrieve your token after it is generated.***
```

Comment:
```suggestion
      **Warning**: Treat your personal access token like your password and keep it secret. You cannot retrieve your token after it is generated.
```

Reply:
AFAIK this is auto rendering and we shouldn't change it. Can check though

Reply:
I think we really want bold and italic to stand out here.